Rock/Model/GroupService.Partial.cs

```diff
@@ -760,7 +760,8 @@ public static Group SaveNewGroup( RockContext rockContext, int groupTypeId, Guid
         /// <param name="country">The country.</param>
         /// <param name="moveExistingToPrevious">if set to <c>true</c> [move existing to previous].</param>
         public static void AddNewGroupAddress( RockContext rockContext, Group group, string locationTypeGuid,
-            string street1, string street2, string city, string state, string postalCode, string country, bool moveExistingToPrevious = false )
+            string street1, string street2, string city, string state, string postalCode, string country, bool moveExistingToPrevious = false,
+            string modifiedBy = "", bool isMailingLocation = true, bool isMappedLocation = true )
```

Comment:
@chrismcgrath We've found that optional parameters on public methods are difficult to maintain backward compatibility. For example, this change would break compatibility, but because of the optional parameter that was already there, there's not really any way around that. Because of that can you change this to use method overrides instead. Like this...
```
        public static void AddNewGroupAddress( RockContext rockContext, Group group, string locationTypeGuid, int? locationId )
        {
            AddNewGroupAddress( rockContext, group, locationTypeGuid, locationId, false );
        }

        public static void AddNewGroupAddress( RockContext rockContext, Group group, string locationTypeGuid, int? locationId, bool moveExistingToPrevious )
        {
            AddNewGroupAddress( rockContext, group, locationTypeGuid, locationId, moveExistingToPrevious, "", true, true );
        }

        public static void AddNewGroupAddress( RockContext rockContext, Group group, string locationTypeGuid, int? locationId, bool moveExistingToPrevious, string modifiedBy, bool isMailingLocation, bool isMappedLocation )
        {
            if ( locationId.HasValue )
            {
                var location = new LocationService( rockContext ).Get( locationId.Value );
                AddNewGroupAddress( rockContext, group, locationTypeGuid, location, moveExistingToPrevious );
            }
        }
```
That will still be a breaking change this time, but if we then need to add additional params in future, we can do it without it being a breaking change.